notebooks/official/pipelines/google_cloud_pipeline_components_bqml_text.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    def pipeline(
</pre><p>the bq queries should be passed in as arguments -- not hardwired.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p>Done</p>